packages/webdoc-default-template/tmpl/layout.tmpl

```diff
@@ -24,10 +24,7 @@ const rootData = obj
     <header class="header" id="header-mount-point"></header>
     <article class="page">
       <div class="page-explorer" id="explorer-mount-point"></div>
-      <div class="page-content">
-        <?js= content ?>
-      </div>
-      <div class="page-members" id="member-mount-point"></div>
+      <?js= content ?>
```

Comment:
Add comment:
```
<?js /* Includes `content` and `member-explorer`. */ ?>
```
JS, because we don't need unnecessary debug comments in html output.

Reply:
Done!